deployment/helm/node-feature-discovery/templates/clusterrole.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.rbac.create }}
+{{- if .Values.master.rbac.create }}
```

Comment:
Add `-master` suffix  to the `ClusterRole` name, like `topologyUpdater`.

Reply:
Done